### DISCUSSION TITLE
Is it possible to partially update an accessory/volume during redeployment?

### DISCUSSION THREAD

**Discussion by charnould** [basecamp/kamal#1268] [Q&A]
I've read carefully GH and haven't found solution to my problem.  

- I have a GH repo I deploy with Kamal perfectly; thanks for this lib.
- In this repo, I have a folder called `datastores` containing 2 `sqlite` files:
  - one is my running DB (called `db.sqlite`)
  - the other is just a static archive of data (called `archive.sqlite`)
- `datastores` is mounted as a `volume` with this config in my `config.yaml`

```
volumes:
  - 'datastores:/app/datastores/'
```

My problem is that sometimes I update `archive.sqlite` **locally**. And when I redeploy I'd love `archive.sqlite` to be up-to-date. However, as Kamal docs say: "[Accessories] are not updated when you deploy".

How can I handle my use case if I'd prefer keep my 2 sqlite files in the same folder?

Thanks a lot.

**Comment by nickhammond** [comment]
@charnould The `files` setting will copy the file from your local machine up to the server but this would always copy it, there's not really a way to _sometimes_ copy it. You could always just write a standalone bash script to rsync your archive.sqlite file up to the server when needed though.

https://kamal-deploy.org/docs/configuration/accessories/#copying-files

What's the workflow/use case around updating `archive.sqlite` on the server vs. local?
